### ISSUE TITLE
[Evaluation] Convert Devin's output into SWE-Bench runnable format

### ISSUE THREAD

**Issue by xingyaoww** [OpenDevin/OpenDevin#80]
We can format [Devin's released output patches](https://github.com/CognitionAI/devin-swebench-results/tree/main/output_diffs) into [SWE-Bench's prediction format](https://github.com/princeton-nlp/SWE-bench/blob/main/tutorials/evaluation.md#-creating-predictions), once that's completed, we can use it to debug the issue of SWE-Bench and make sure **we can reproduce the results from Devin**.

```python
{
    "instance_id": "<Unique task instance ID>",
    "model_patch": "<.patch file content string>",
    "model_name_or_path": "<Model name here (i.e. SWE-Llama-13b)>",
}
```

**Comment by Jiaxin-Pei** [comment]
created a [PR](https://github.com/OpenDevin/OpenDevin/pull/81) for this issue

**Comment by xingyaoww** [comment]
Addressed by #81
